pkg/controller/node/nodecontroller.go

```diff
@@ -371,6 +402,42 @@ func (nc *NodeController) Run() {
 			}
 		}, nc.nodeMonitorPeriod, wait.NeverStop)
 
+		go wait.Until(func() {
+			nc.evictorLock.Lock()
```

Comment:
To make it a bit more specific. The flow will be: PodEvictor holds Nodes to be evicted and both IntolerablePodEvictor and TerminationEvictor will hold Pods to be evicted.